script.js

```diff
@@ -1,8 +1,19 @@
+console.log("hello")
+
+$(document).ready(function() {
+
+})
 // Create an array called `colors` that contains three colors.
 
+var colors = ["green", "yellow", "red"];
+
 // Log the last item in the array to the console.
+console.log(colors[2])
 
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
+for (i = 0; i < colors.length; i++) {
+  console.log(colors[i])
+}
 
```

Comment:
you'd just have to add `"My favorite color is: " + colors[i]` inside `console.log()`